src/server/modelManager.js

```diff
@@ -1,5 +1,13 @@
+const Grid = require("./models/grid");
+
+var g;
+
 function createGrid(json)
 {
-
+    g=new Grid(json.nbKeel,json.nbFrame);
```

Comment:
Maybe create variables for each extracted parameters from the json. It will be easier to read and debug.

Reply:
done